Build dashboard from scratch failed

#### Issue details

##### Environment
<!-- Describe how do you run Kubernetes and Dashboard. 
     Versions of Node.js, Go etc. are needed only from developers. To get them use console:
     $ node --version
     $ go version
 -->

```
Dashboard version:
Kubernetes version: 
Operating system: Ubuntu 16.04
Node.js version: v6.10.0
Go version: go1.6.2 linux/amd64
```

##### Steps to reproduce
<!-- Describe all steps needed to reproduce the issue. It is a good place to use numbered list. -->
Followed steps described in [readme,](https://github.com/kubernetes/dashboard/blob/master/docs/devel/getting-started.md) when executing last step:
 gulp serve/gulp build both fails
##### Observed result
<!-- Describe observed result as precisely as possible. -->
[01:22:40] Finished 'generate-xtbs' after 14 μs
[01:22:40] Starting 'scripts:prod'...
.tmp/backend/src/github.com/kubernetes/dashboard/vendor/k8s.io/client-go/rest/request.go:21:2: cannot find package "context" in any of:
        /home/lvroyce/work/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/context (vendor tree)
        /usr/local/go/src/context (from $GOROOT)
        /home/lvroyce/work/dashboard/.tmp/backend/src/context (from $GOPATH)
        /home/lvroyce/work/dashboard/vendor/src/context
[01:22:44] 'backend:prod' errored after 26 s
[01:22:44] Error: Go command error, code: 1
    at Error (native)
    at ChildProcess.<anonymous> (/home/lvroyce/work/dashboard/build/gocommand.js:172:23)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
[01:23:42] Finished 'scripts:prod' after 1.03 min
##### Expected result
<!-- Describe expected result as precisely as possible. -->
Build success
##### Comments
<!-- If you have any comments or more details, put them here. -->

--- a/docs/devel/getting-started.md
+++ b/docs/devel/getting-started.md
@@ -14,7 +14,7 @@ logic and fetches raw data from the various Kubernetes APIs.
 Make sure the following software is installed and added to the `$PATH` variable:
 
 * Docker (1.10+)
-* go (1.6.1+)
+* go (1.7+)
 * nodejs (5.1.1+)
 * npm (3+)
 * java (7+)

